marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -263,12 +264,28 @@ function getEntityInstance(docUri) {
 }
 
 function getEntityInstanceHeaders(docUri) {
-    let doc = cts.doc(docUri);
+  const doc = cts.doc(docUri);
```

Comment:
What's the actual requirement here? The JIRA ticket mentions that "headers are not extracted", and this function is called "getEntityInstanceHeaders", but the code is specifically interested in the sources part of headers. 

What is the UI looking to do with the sources? 

If the intent is to provide an array of sources for an entity instance, I think this function should be called "getEntitySources" (I think the word 'instance' adds a little confusion because 'instance' and 'headers' are two separate parts of the envelope).

Reply:
Sources are displayed in the instance and snippet view in the Explorer tile

Updated the function name to getEntitySources as suggested